resident/resident-service/src/main/java/io/mosip/resident/util/TemplateUtil.java

```diff
@@ -201,19 +197,32 @@ private String getAttributesDisplayText(String attributesFromDB, String language
 		}
 	}
 
-	private String getAuthTypeCodeTemplateValue(String authenticationMode, String languageCode) {
-		return getTemplateValueFromTemplateTypeCodeAndLangCode(languageCode,
-				AuthenticationModeEnum.getTemplatePropertyName(authenticationMode, env));
-	}
-
 	public String getTemplateValueFromTemplateTypeCodeAndLangCode(String languageCode, String templateTypeCode) {
 		return proxyMasterdataService
 					.getTemplateValueFromTemplateTypeCodeAndLangCode(languageCode, templateTypeCode);
 	}
 
 	public String getDescriptionTemplateVariablesForAuthenticationRequest(
 			ResidentTransactionEntity residentTransactionEntity, String fileText, String languageCode) {
-		return residentTransactionEntity.getStatusComment();
+		List<String> authTypeCodeTemplateValues = new ArrayList<>();
+		String statusCode = residentService.getEventStatusCode(residentTransactionEntity.getStatusCode(), languageCode)
+				.getT1();
+		String authTypeCodeFromDB = residentTransactionEntity.getAuthTypeCode();
+		if (authTypeCodeFromDB != null && !authTypeCodeFromDB.isEmpty()) {
+			List<String> authTypeCodeListFromDB = List
```

Comment:
Use Stream API to map it to the new list, avoid creating separate list and adding elements in for loop.

Reply:
done